src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,11 @@
+@keyframes move {
+  0% {
+    transform: rotate(0deg);
+    transform-origin: bottom center;
```

Comment:
remove this rule from animation and just add to minute/second hands 
`transform-origin: bottom center;`